File: 2572-append-characters-to-string-to-make-subsequence/solution.java
class Solution {
    public int appendCharacters(String s, String t) {
        int i=0,j=0;
        int n=t.length();
        int m=s.length();
        while(i<m && j<n)
        {
            if(s.charAt(i)==t.charAt(j))
            {
                j++;
            }
            i++;
        }
        return n-j;
    }
}

File: 2572-append-characters-to-string-to-make-subsequence/solution.c
int appendCharacters(char* s, char* t) {
    int i=0,j=0;
    int n=strlen(t);
    while(s[i]!='\0')
    {
        if(s[i]==t[j])
        {
            j++;
        }
        i++;
    }
    return n-j; 
}
